### DISCUSSION TITLE
Translation issues

### DISCUSSION THREAD

**Discussion by refugeindhamma** [liedekef/events-made-easy#317] [Q&A]
Hi Franky,

I'm using Polylang and Loco Translate.

1) I have checked that the "Events" Page has no language assigned.
2) The registration form is very erratic in loading the correct language. Most of the time English is displayed.
3) When the registration form is displayed in Chinese, I get this "Registration Confirmed" email that is half in Chinese, half in English. I have checked my templates and there are no issues there. As can be seen in the screenshot below, the custom fields and name of sender are shown in Chinese correctly while the rest of the email is in English. _(Title of the event is written in Chinese so there is no translation there.)_
![image](https://github.com/liedekef/events-made-easy/assets/101639308/672b6959-e340-4d47-b17d-9c01dff1a9fb)

Edit:
This is my template:
```
[:en]
Dear #_FIRSTNAME,

You have successfully registered #_RESPSEATS seat(s) for #_EVENTNAME.

The details are as follows:
<BR />
#_PERSONFULLNAME - #_FIELDVALUE{8}<BR />
#_DYNAMICFIELD{4} - #_DYNAMICFIELD{9}<BR />
#_DYNAMICFIELD{5} - #_DYNAMICFIELD{10}<BR />
#_DYNAMICFIELD{6} - #_DYNAMICFIELD{11}<BR />
#_DYNAMICFIELD{7} - #_DYNAMICFIELD{12}
<BR /><BR />
<strong>PLEASE READ THE ATTACHED PDF FOR DETAILS OF THE MEDITATION RETREAT.</strong>
<BR /><BR />
With Metta,
#_CONTACTPERSON
[:zh]
亲爱的 #_FIRSTNAME,

你已经成功预订了 #_EVENTNAME 的 #_RESPSEATS 个座位。

详情如下：
<BR />
#_PERSONFULLNAME - #_FIELDVALUE{8}<BR />
#_DYNAMICFIELD{4} - #_DYNAMICFIELD{9}<BR />
#_DYNAMICFIELD{5} - #_DYNAMICFIELD{10}<BR />
#_DYNAMICFIELD{6} - #_DYNAMICFIELD{11}<BR />
#_DYNAMICFIELD{7} - #_DYNAMICFIELD{12}
<BR /><BR />
<strong>请阅读随附的 PDF 文件以了解禅修营的详细信息。</strong>
<BR /><BR />
祝安康快乐，
#_CONTACTPERSON
[:]
```

Any idea what went wrong?

**Comment by liedekef** [comment]
How is polylang configured? It can be that EME is not correctly chosing the language.

**Reply by refugeindhamma** [reply]
URL settings in polylang:
![Screenshot 2023-10-11 115207](https://github.com/liedekef/events-made-easy/assets/101639308/fdefc95d-60ec-43d5-b06a-dbc1b4371ad1)

The settings in polylang have not changed since I installed it and got it working.

**Reply by liedekef** [reply]
Checked it here: works fine with polylang and chinese and a user doing a booking.
Also: if posting an issue, go to "issues", not open a discussion. (Can't move this to issues, so please open an issue if the problem still exists).

**Reply by refugeindhamma** [reply]
I found out the issue. It's the cache that is interfering with the translation. 

Thanks for your help.
